src/FakeItEasy/Configuration/RuleBuilder.cs

```diff
@@ -37,14 +37,11 @@ internal RuleBuilder(BuildableCallRule ruleBeingBuilt, FakeManager manager, Fake
 
         public ICallMatcher Matcher => new RuleMatcher(this);
 
-        private RuleBuilder Then
-        {
-            get
+        private RuleBuilder Then =>
+            new RuleBuilder(this.RuleBeingBuilt.CloneCallSpecification(), this.manager, this.asserterFactory)
             {
-                var newRule = this.RuleBeingBuilt.CloneCallSpecification();
-                return new RuleBuilder(newRule, this.manager, this.asserterFactory) { PreviousRule = this.RuleBeingBuilt };
-            }
-        }
+                PreviousRule = this.RuleBeingBuilt
+            };

```

Comment:
In this case I find the non-expression-bodied version easier to read. The single expression does a bit too many things for my taste.